extension/src/popup/views/Wallets/hooks/useGetWalletsData.tsx

```diff
@@ -147,7 +147,10 @@ function useGetWalletsData() {
       dispatch({ type: "FETCH_DATA_SUCCESS", payload });
       return payload;
     } catch (error) {
-      dispatch({ type: "FETCH_DATA_ERROR", payload: error });
+      dispatch({
+        type: "FETCH_DATA_ERROR",
+        payload: { error, isFetchingTokenPrices: false },
```

Comment:
does the payload here accept both formats?

`payload: error`
and 
`payload: { error, isFetchingTokenPrices: false }`